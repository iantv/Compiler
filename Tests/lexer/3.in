sstr1 abc123
123abc int1 double23