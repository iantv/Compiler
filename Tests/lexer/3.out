1	1	sstr1		identifier
1	7	abc123		identifier
2	1	123		int_val
2	4	abc		identifier
2	8	int1		identifier
2	13	double23	identifier
